src/command_modules/azure-cli-interactive/azclishell/_dump_commands.py

```diff
@@ -55,33 +58,36 @@ def dump_command_table():
     for cmd in CMD_TABLE:
         com_descrip = {}
         param_descrip = {}
-        command_description = CMD_TABLE[cmd].description
-        if callable(command_description):
-            command_description = command_description()
-        com_descrip['help'] = command_description
-        com_descrip['examples'] = ""
-
-        for key in CMD_TABLE[cmd].arguments:
-            required = ""
-            help_desc = ""
-            if CMD_TABLE[cmd].arguments[key].type.settings.get('required'):
-                required = "[REQUIRED]"
-            if CMD_TABLE[cmd].arguments[key].type.settings.get('help'):
-                help_desc = CMD_TABLE[cmd].arguments[key].type.settings.get('help')
-
-            name_options = []
-            for name in CMD_TABLE[cmd].arguments[key].options_list:
-                name_options.append(name)
-
-            options = {
-                'name': name_options,
-                'required': required,
-                'help': help_desc
-            }
-            param_descrip[CMD_TABLE[cmd].arguments[key].options_list[0]] = options
-
-        com_descrip['parameters'] = param_descrip
-        data[cmd] = com_descrip
+        try:
+            command_description = CMD_TABLE[cmd].description
+            if callable(command_description):
+                command_description = command_description()
+            com_descrip['help'] = command_description
+            com_descrip['examples'] = ""
+
+            for key in CMD_TABLE[cmd].arguments:
+                required = ""
+                help_desc = ""
+                if CMD_TABLE[cmd].arguments[key].type.settings.get('required'):
+                    required = "[REQUIRED]"
+                if CMD_TABLE[cmd].arguments[key].type.settings.get('help'):
+                    help_desc = CMD_TABLE[cmd].arguments[key].type.settings.get('help')
+
+                name_options = []
+                for name in CMD_TABLE[cmd].arguments[key].options_list:
+                    name_options.append(name)
+
+                options = {
+                    'name': name_options,
+                    'required': required,
+                    'help': help_desc
+                }
+                param_descrip[CMD_TABLE[cmd].arguments[key].options_list[0]] = options
+
+            com_descrip['parameters'] = param_descrip
+            data[cmd] = com_descrip
+        except (ImportError, ValueError):
+            continue
```

Comment:
Would suggest `pass` instead of `continue` here.
https://stackoverflow.com/questions/9483979/is-there-a-difference-between-continue-and-pass-in-a-for-loop-in-python